packages/host/app/services/store.ts

```diff
@@ -166,13 +166,9 @@ export default class StoreService extends Service implements StoreInterface {
     }
   }
 
-  addReference(idOrDoc: string | SingleCardDocument | undefined) {
```

Comment:
this was confusing that `Store.addReference()` could also take in a document--cleaned this up